### ISSUE TITLE
Performance Issue with FeCAM_cifar100.json in Many-Shot CIL Scenarios

### ISSUE THREAD

**Issue by liuruiqi0520** [dipamgoswami/FeCAM#5]
I used FeCAM_cifar100.json for many-shot CIL scenarios with/without pre-trained ResNet18 and achieved 62% average top-1 incremental accuracy. Is this expected, or could it indicate a mistake in my implementation? Thank you for your guidance.

**Comment by dipamgoswami** [comment]
Hi, 
Without using pre-trained weights, it is expected to get an accuracy close to 62% after the last task.
If you are using pre-trained weights, the accuracy is expected to be better. To do so, you need to set pretrained option to True
`self._network = CosineIncrementalNet(args, True)` in models/fecam.py line 26. 

The accuracy after the first task should at least be better. Try to change the shrinkage values and see if that makes a difference. If this doesn't work, let me know. I will try it on my end.
